diff --git a/media.module b/media.module
index 1d4c9cf..187e704 100644
--- a/media.module
+++ b/media.module
@@ -359,7 +359,8 @@ function media_form_field_ui_field_edit_form_alter(&$form, &$form_state) {
 
   // Add a validation function to any field instance which uses the media widget
   // to ensure that the upload destination scheme is one of the allowed schemes.
-  if ($form['instance']['widget']['type']['#value'] == 'media_generic') {
+  // Multimedia asset fields settings is not validating.
+  if ($form['instance']['widget']['type']['#value'] == 'media_generic' && $form['#field']['type'] !== 'media') {
     $form['#validate'][] = 'media_field_instance_validate';
   }
 
