diff --git a/media.module b/media.module
index 1d4c9cf..ecf7cb2 100644
--- a/media.module
+++ b/media.module
@@ -379,7 +379,8 @@ function media_form_field_ui_field_edit_form_alter(&$form, &$form_state) {
  */
 function media_field_instance_validate($form, &$form_state) {
   $allowed_schemes = $form_state['values']['instance']['widget']['settings']['allowed_schemes'];
-  $upload_destination = $form_state['values']['field']['settings']['uri_scheme'];
+  $upload_destination = !empty( $form_state['values']['field']['settings']['uri_scheme']) ?
+    $form_state['values']['field']['settings']['uri_scheme'] : file_default_scheme();
 
   if (!in_array($upload_destination, array_filter($allowed_schemes))) {
     form_set_error('allowed_schemes', t('The upload destination must be one of the allowed schemes.'));

