=== modified file 'linkedin_profile/linkedin-user-page-education-item.tpl.php'
--- old/linkedin_profile/linkedin-user-page-education-item.tpl.php	2012-02-27 00:42:00 +0000
+++ new/linkedin_profile/linkedin-user-page-education-item.tpl.php	2012-02-27 03:31:43 +0000
@@ -13,17 +13,17 @@

   <p>
     <?php print $education['degree'] ?>
-    <?php if ($education['field-of-study']) : ?>
+    <?php if (isset($education['field-of-study'])) : ?>
       <?php print t('in') . ' ' . $education['field-of-study']; ?>
     <?php endif; ?>
-    <?php if ($education['school-name']) : ?>
+    <?php if (isset($education['school-name'])) : ?>
       <?php print t('at') . ' ' . $education['school-name']; ?>
     <?php endif; ?>
-    <?php if($education['start-date']['year'] || $education['end-date']['year']) : ?>
+    <?php if(isset($education['start-date']['year']) || isset($education['end-date']['year'])) : ?>
       <span class="linkedin-education-years">
       <?php print ' ('; ?>
       <?php print $education['start-date']['year']; ?>
-        <?php if($education['start-date']['year'] && $education['end-date']['year']) : ?>
+        <?php if(isset($education['start-date']['year']) && isset($education['end-date']['year'])) : ?>
           <?php print '-'; ?>
         <?php endif; ?>
       <?php print $education['end-date']['year']; ?>

=== modified file 'linkedin_profile/linkedin-user-page.tpl.php'
--- old/linkedin_profile/linkedin-user-page.tpl.php	2012-02-27 00:42:00 +0000
+++ new/linkedin_profile/linkedin-user-page.tpl.php	2012-02-27 03:29:49 +0000
@@ -12,13 +12,13 @@
 ?>
 <div class="linkedin-profile">

-  <?php if ($profile['picture-url']) : ?>
+  <?php if (isset($profile['picture-url'])) : ?>
     <a href="<?php print $profile['public-profile-url']['value']; ?>" <?php if ($profile['first-name']['value'] || $profile['last-name']['value']) : ?> title="Public profile of <?php print $profile['first-name']['value'] . ' ' . $profile['last-name']['value']; ?> on www.linkedin.com."<?php endif; ?> >
         <img src="<?php print $profile['picture-url']['value']; ?>" alt="<?php print $profile['first-name']['value'] . ' ' . $profile['last-name']['value']; ?>" />
       </a>
   <?php endif; ?>

-  <?php if ($profile['first-name']['value'] || $profile['last-name']['value']) : ?>
+  <?php if (isset($profile['first-name']['value']) || isset($profile['last-name']['value'])) : ?>
         <a href="<?php print $profile['public-profile-url']['value']; ?>" title="Public profile of <?php print $profile['first-name']['value'] . ' ' . $profile['last-name']['value']; ?> on www.linkedin.com.">
           <h2 class="linkedin-name"><?php print $profile['first-name']['value'] . ' ' . $profile['last-name']['value']; ?></h2>
         </a>
@@ -26,7 +26,7 @@

         <ul>
     <?php foreach ($profile as $key => $value) : ?>
-    <?php if ($value && !in_array($key, array ('picture-url', 'first-name', 'last-name'))) : ?>
+    <?php if (isset($value) && !in_array($key, array ('picture-url', 'first-name', 'last-name'))) : ?>
             <li class="linkedin-<?php print $key; ?>"><?php print $value['name']; ?> : <?php print $value['value']; ?>
       <?php endif; ?>
       <?php endforeach; ?>

=== modified file 'linkedin_profile/linkedin_profile.module'
--- old/linkedin_profile/linkedin_profile.module	2012-02-27 02:06:20 +0000
+++ new/linkedin_profile/linkedin_profile.module	2012-02-27 03:26:32 +0000
@@ -113,7 +113,7 @@
             $items = array();
             $items_val = $profile[$key][$multiples[$key]];
             if (is_array($items_val)) {
-              if ($items_val['id']) {
+              if (isset($items_val['id'])) {
                 $variables[$multiples[$key]] = $items_val;
                 $items[] = theme('linkedin_profile_user_page_' . $multiples[$key] . '_item', $variables);
               }
